.github/actions/setup/action.yml

```diff
@@ -3,15 +3,15 @@ description: Install dependencies and perform setup for https://github.com/arkty
 
 inputs:
     node:
-        default: lts/*
+        default: latest
```

Comment:
I think it makes sense for the default to remain the latest lts build, with `latest` still be run as one of the matrix tests.

Reply:
You're right, we don't necessarily need this with `latest` in the pr matrix.